mne/io/brainvision/brainvision.py

```diff
@@ -648,7 +658,7 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
 
     # locate EEG binary file and marker file for the stim channel
     path = op.dirname(vhdr_fname)
-    data_filename = op.join(path, cfg.get(cinfostr, 'DataFile'))
+    data_fname = op.join(path, cfg.get(cinfostr, 'DataFile'))
```

Comment:
why are changing the variable name? I mean, I'm ok with both. Do we have a naming convention or could we start one?

Reply:
this was for two reasons:
1. it's shorter
2. it's consistent with `marker_fname`, which we were using before